presto-main/src/main/java/io/prestosql/server/HttpRequestSessionContext.java

```diff
@@ -208,6 +208,17 @@ public static Identity extractAuthorizedIdentity(Optional<Identity> optionalAuth
         return identity;
     }
 
+    /**
+     * Creates a session identity.
+     * <p>
+     * Checks if the user is non-null, add header, add extraCredentials and add groups to build a session identity.
```

Comment:
Your description should be more like a text, and not just steps.
I would do this description like this:
"Create a Session Identity according to the defined parameters headers, credentials and groups."